pkg/support/dump.go

```diff
@@ -293,7 +297,21 @@ func (d *agentDumper) DumpAgentInfo(basedir string) error {
 }
 
 func (d *agentDumper) DumpMemberlist(basedir string) error {
-	return dumpAntctlGet(d.fs, d.executor, "memberlist", basedir)
+	handler := memberlist.HandleFunc(d.aq)
+	req, err := http.NewRequest(http.MethodGet, "", nil)
+	if err != nil {
+		return err
+	}
+	recorder := httptest.NewRecorder()
+	handler.ServeHTTP(recorder, req)
+	var received []memberlist.Response
+	if recorder.Code == http.StatusOK {
+		err = json.Unmarshal(recorder.Body.Bytes(), &received)
+		if err != nil {
+			return err
+		}
+	}
+	return writeYAMLFile(d.fs, filepath.Join(basedir, "memberlist"), "memberlist", received)
```

Comment:
Why is it still needed?

Reply:
because windows tests fail with same reason if we keep existing code (even after making changes in pkg/agent/apiserver/handlers/memberlist/handler.go to check for memberlist is running or not). I am not aware of the reason for this but i guess there maybe some history of running antctl cmds  on windows as I see  `DumpAgentInfo` func in dump.go also uses similar approach instead of calling `dumpAntctlGet` function.

Reply:
Got it, thanks.

Reply:
@tnqn Do you know the reason for `Use internal queriers instead of antctl to
retrieve networkpolicies and agentinfo.` ? as written in commit message of this PR https://github.com/antrea-io/antrea/pull/820